sunbird-rc-ED-ui/proxy.conf.json

```diff
@@ -1,21 +1,21 @@
 {
     "/registry/api/v1": {
-      "target": "https://dev-eg2.uniteframework.io",
+      "target": "https://eg2.uniteframework.io",
```

Comment:
@code-blooded-guy Remove this file from merge

Reply:
This file will not reflect anywhere in deployment, it's only for local development